packages/web/src/components/molecules/Console/Console.tsx

```diff
@@ -193,7 +195,7 @@ export const Console = forwardRef<ConsoleRef, ConsoleProps>(({content, wrap, Lin
   }, [viewportStart, viewportEnd, scrollTop, clientHeight, total]);
 
   // Re-render on scroll
-  const rerenderDebounce = useMemo(() => debounce(rerender, 5), []);
+  const rerenderDebounce = useMemo(() => debounce(rerender, 5), [rerender]);
```

Comment:
Does `rerender` persist the identity, or the `rerenderDebounce` is recalculated every render?

Reply:
It persist and it is not recalculated on every rerender